### PR TITLE
Fix sending snowplow events for dashboard tabs

### PR DESCRIPTION
**Pull request by ranquild**
Currently snowplow events for dashboard tabs are broken because of the typo in the event name. This PR fixes the issue and merges `dashboardtab` schema into `dashboard` schema to follow the existing convention (this is done to unlock querying data in redshift). 

How to test:
- New cypress test should pass

### CODE REVIEW

File: snowplow/iglu-client-embedded/schemas/com.metabase/dashboard/jsonschema/1-1-0 [hunk 1]
```diff
@@ -1,47 +1,63 @@
 {
   "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
-  "description": "Dashboard tab events",
+  "description": "Dashboard events",
```
Comment (ranquild) [inline snowplow/iglu-client-embedded/schemas/com.metabase/dashboard/jsonschema/1-1-0:3]: This is the copy of the previous `dashboard` schema (version `1-0-1`) with new properties from `dashboardtab` with `null` types.

File: src/metabase/api/dashboard.clj [hunk 2]
```diff
@@ -616,15 +616,15 @@
 
   ;; Tabs events
   (when (seq deleted-tab-ids)
-    (snowplow/track-event! ::snowplow/dashboard-tabs-deleted
+    (snowplow/track-event! ::snowplow/dashboard-tab-deleted
```
Comment (ranquild) [inline src/metabase/api/dashboard.clj:619]: The source of the bug

File: src/metabase/analytics/snowplow.clj [hunk 3]
```diff
@@ -135,14 +135,13 @@
 (def ^:private schema->version
   "The most recent version for each event schema. This should be updated whenever a new version of a schema is added
   to SnowcatCloud, at the same time that the data sent to the collector is updated."
-  {::account      "1-0-0"
+  {::account      "1-0-1"
```
Comment (ranquild) [inline src/metabase/analytics/snowplow.clj:138]: Some event versions were outdated

File: e2e/test/scenarios/dashboard/tabs.cy.spec.js [hunk 4]
```diff
@@ -58,3 +63,33 @@ describe("scenarios > dashboard tabs", () => {
     });
   });
 });
+
+describeWithSnowplow.only("scenarios > dashboard > tabs", () => {
+  const PAGE_VIEW_EVENT = 1;
```
Comment (ranquild) [inline e2e/test/scenarios/dashboard/tabs.cy.spec.js:68]: This test looks like others for snowplow events

**Review by qnkhuat — APPROVED** [review]
The change lgtm, thanks for fixing this.

Note that we also need to delete the schema and table on redshift.

**Pull request merged**